device/mellanox/x86_64-mlnx_msn3420-r0/sensors.conf

```diff
@@ -33,8 +33,8 @@ bus "i2c-5" "i2c-1-mux (chan_id 4)"
         label curr1 "PMIC-1 COMEX 1.8V Rail Curr (out)"
         label curr2 "PMIC-1 COMEX 1.05V Rail Curr (out)"
     chip "tps53679-i2c-*-71"
-        label in1 "PMIC-2 PSU 12V Rail (in)"
-        label in2 "PMIC-2 PSU 12V Rail (in)"
+        label in1 "PMIC-2 PSU 12V Rail (in1)"
+        label in2 "PMIC-2 PSU 12V Rail (in2)"
```

Comment:
@shlomibitton why do we need in1/in2. IMHO, it doesn't provide to much info for user. What was the motivation for renaming? Please elaborate.

Reply:
on sonic-mgmt we can't have the same label for both in1 and in2. Doing so will fail the sensor output parser, so we have to change the labels like this.
to be consistent and match the labels I have changed them on sonic-buildimage as well.

Reply:
@shlomibitton thanks for explanation.